include/LLVMSPIRVLib.h

```diff
@@ -50,19 +50,19 @@ namespace llvm {
 // Pass initialization functions need to be declared before inclusion of
 // PassSupport.h.
 class PassRegistry;
-void initializeLLVMToSPIRVPass(PassRegistry &);
-void initializeOCLToSPIRVPass(PassRegistry &);
-void initializeOCLTypeToSPIRVPass(PassRegistry &);
-void initializeSPIRVLowerBoolPass(PassRegistry &);
-void initializeSPIRVLowerConstExprPass(PassRegistry &);
-void initializeSPIRVLowerSPIRBlocksPass(PassRegistry &);
-void initializeSPIRVLowerOCLBlocksPass(PassRegistry &);
-void initializeSPIRVLowerMemmovePass(PassRegistry &);
-void initializeSPIRVLowerSaddWithOverflowPass(PassRegistry &);
-void initializeSPIRVRegularizeLLVMPass(PassRegistry &);
-void initializeSPIRVToOCL12Pass(PassRegistry &);
-void initializeSPIRVToOCL20Pass(PassRegistry &);
-void initializePreprocessMetadataPass(PassRegistry &);
+void initializeLLVMToSPIRVLegacyPass(PassRegistry &);
```

Comment:
I was always wondering why do we have these declarations in `LLVMSPIRVLib.h`? Do we really provide those as part of our public interface?

But this is for sure not a problem of this PR, just a question/observation.